### PR TITLE
User dropdown menu

### PR DESCRIPTION
**Pull request by 105ron**
This is for [issue 590](https://github.com/TheOdinProject/theodinproject/issues/590). There is no mobile responsive menu yet.

### CODE REVIEW

**Comment by 105ron** [comment]
I did not mean to remove facebook-icon.svg . I'll add it in on the next commit🙀

**Comment by KevinMulhern** [comment]
👍 This is looking so good
Thanks for putting it together @105ron 🎉

**Comment by arun1595** [comment]
@105ron Looks great :heart_eyes:

File: app/views/shared/_user_dropdown.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="dropdown">
+  <div class="avatar" style="background-image:url(<%= current_user.image %>)" data-toggle="dropdown"></div>
+  <ul class="dropdown-menu menu-options">
+    <li>
+      <%= link_to raw('<span class="drop-down-logo dashboard-logo"></span>Dashboard'), dashboard_path, class: 'drop-down-link accent' %>
```
Comment (arun1595) [inline app/views/shared/_user_dropdown.html.erb]: `link_to` accepts a block and the link target HTML can be passed in the block.

```
<%= link_to dashboard_path, class: 'drop-down-link accent'  do %>
  <span class="drop-down-logo dashboard-logo"></span>Dashboard
<% end %>
```

File: app/assets/images/icons/icon-facebook.svg [hunk 2]
```diff
@@ -0,0 +1,841 @@
+
+
+
+
+
+
+<!DOCTYPE html>
```
Comment (KevinMulhern) [inline app/assets/images/icons/icon-facebook.svg]: This file doesn't seem right @105ron, it looks like the source code for the home page of the site?

File: app/assets/images/icons/icon-facebook.svg [hunk 3]
```diff
@@ -1,5 +1,841 @@
...
-        <tspan x="147.694" y="22"></tspan>
-    </text>
-</svg>
+
+
+
+
+
+
+<!DOCTYPE html>
```
Comment (KevinMulhern) [inline app/assets/images/icons/icon-facebook.svg]: This file doesn't look right @105ron, it looks like the source code for the home page of the site?

**Comment by KevinMulhern** [comment]
👍 LGTM
🎉 Thank you Rhys

**Pull request merged**